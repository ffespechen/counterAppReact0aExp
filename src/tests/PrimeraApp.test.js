import React from 'react';
import PrimeraApp from '../PrimeraApp';
import '@testing-library/jest-dom';
// import { render } from '@testing-library/react';
import { shallow } from 'enzyme';

describe('Pruebas componentes <PrimeraApp />', () =>{

	test('DeberÃ­a mostrar "Hola soy Ironman!"', () =>{

		const mensaje = 'Hola soy Ironman!';

		// Desestructuro el resultado, en lugar de obtener una const
		// const { getByText } = render( <PrimeraApp saludo={ mensaje }/>);

		// En el expect verifico si tengo el texto esperado
		// expect( getByText(mensaje) ).toBeInTheDocument();

		const wrapper = shallow( <PrimeraApp saludo={ mensaje }/> );

		expect( wrapper ).toMatchSnapshot();

	});

});