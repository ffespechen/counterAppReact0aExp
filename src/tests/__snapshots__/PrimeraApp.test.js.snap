// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas componentes <PrimeraApp /> Deber√≠a mostrar "Hola soy Ironman!" 1`] = `
<Fragment>
  <h1>
     
    Hola soy Ironman!
     
  </h1>
  <p>
     
    Soy un subtitulo
     
  </p>
</Fragment>
`;
